.github/workflows/build-iroha1-fork.yml

```
@@ -421,22 +426,22 @@ jobs:
           echo >>$GITHUB_ENV _CCACHE_DIR=$($(realpath $CCACHE_PATH/gcc) --show-config | sed -nE 's,.*cache_dir = ,,p')
           echo >>$GITHUB_ENV NPROC=$(nproc | awk '{printf("%.0f",$1*0.77)}')
           echo >>$GITHUB_ENV HOME=$HOME
-      # - name: Restore cache CCache
-      #   uses: actions/cache@v2
-      #   with:
-      #     path: ${{ env._CCACHE_DIR }}
-      #     key: ${{runner.os}}-ccache-${{ github.event.pull_request.head.sha }}
-      #     restore-keys: ${{runner.os}}-ccache-
-      # - run: ccache --show-stats | tee /tmp/ccache-stats
-      # - ## Read the docs https://vcpkg.readthedocs.io/en/latest/users/binarycaching/ https://github.com/microsoft/vcpkg/blob/master/docs/users/binarycaching.md
-      #   name: Restore cache Vcpkg binarycache ## NOTE not useng NuGet because on ubuntu nuget needs mono of 433MB, unusable.
-      #   uses: actions/cache@v2
-      #   with:
-      #     path: |
-      #       ${{env.HOME}}/.cache/vcpkg/archives
-      #     key: ${{runner.os}}-vcpkg-${{env.CC_NAME}}.${{ hashFiles('vcpkg/**') }}
-      #     restore-keys: ${{runner.os}}-vcpkg-${{env.CC_NAME}}.
-      - name: Build iroha vcpkg dependencies
+      - name: Restore cache CCache
+        uses: actions/cache@v2
+        with:
+          path: ${{ env._CCACHE_DIR }}
+          key: ${{runner.os}}-ccache-${{ github.event.pull_request.head.sha }}
+          restore-keys: ${{runner.os}}-ccache-
+      - run: ccache --show-stats | tee /tmp/ccache-stats
+      - ## Read the docs https://vcpkg.readthedocs.io/en/latest/users/binarycaching/ https://github.com/microsoft/vcpkg/blob/master/docs/users/binarycaching.md
+        name: Restore cache Vcpkg binarycache ## NOTE not useng NuGet because on ubuntu nuget needs mono of 433MB, unusable.
+        uses: actions/cache@v2
+        with:
+          path: |
+            ${{env.HOME}}/.cache/vcpkg/archives
+          key: ${{runner.os}}-vcpkg-${{env.CC_NAME}}.${{ hashFiles('vcpkg/**') }}
+          restore-keys: ${{runner.os}}-vcpkg-${{env.CC_NAME}}.
+      - name: Build iroha vcpkg dependancies
```

```suggestion
      - name: Build iroha vcpkg dependencies
```
Similalrly throughout the workflow.